metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/powerbi_api.py

```diff
@@ -120,11 +123,11 @@ def _get_entity_users(
                 entity=entity_name,
                 entity_id=entity_id,
             )
-        except Exception as e:
-            self.log_http_error(
-                message=f"Unable to fetch users for {entity_name}({entity_id}).", e=e
+        except:  # It will catch all type of exception
```

Comment:
What was the problem with the `Exception as e` way of catching exception handling?

Reply:
+1 - I'm somewhat confused

Reply:
It is not catching ConnectionError, so Exception is not good if you want to catch all types of exceptions.

Reply:
Ah - thanks for clarification. This is useful to know!